src/screens/Home/header.js

```diff
@@ -10,7 +10,7 @@ const Header = () => {
         <img src={ logo } alt="NIMBLE" className="home-header__site-logo" />
       </div>
       <div className="home-header__menu">
-        <img src={ userImage } alt="USER" className="home-header__user-image" />
+        <Sidebar />
```

Comment:
@rafayet-monon Should we make our sidebar more standalone?
This way seems stick with header 🤔

![Screen Shot 2564-01-04 at 12 57 09](https://user-images.githubusercontent.com/749672/103505240-5d249a80-4e8c-11eb-9b6b-4106053cb8d4.png)

Reply:
@rafayet-monon actually what i mean, I think about the structure like this.
```
<div class="home-header">
  <div class="home-header_logo">...</div>
  <div class="home-header_user">...</div>
</div>
<div class="sidebar">
  <div class="sidebar-information">...</div>
  <div class="sidebar-menu">...</div>
</div>
```
(the naming above just for example)

Once we click on `home-header_user`, it will dispatch an event to open `sidebar`.
What do you think? 🤔

Reply:
@Lahphim Implemented in [ca09a6f](https://github.com/rafayet-monon/surveyor/pull/24/commits/ca09a6f41f1e9c2a78d034f7ce09574d8c8901dc) How does this one look?